Refresh.GameServer/Types/Reviews/ReviewRating.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Refresh.GameServer.Types.Reviews;
+
+public class ReviewRating
+{
+    public int PostiveRating { get; set; } = 0;
+    public int NegativeRating { get; set; } = 0;
+}
```

Comment:
I actually like this solution, to the point where I want to refactor a few things across the codebase to better implement this.

Could use a couple things to achieve that:
- Use a generic name, something akin to `DatabaseRatings` (similar to `DatabaseList`)
- XMLDoc to explain the usecase

Reply:
does ObjectRating sound good?

Reply:
nevermind